tests/bluetooth/bsim_bt/bsim_test_mesh/src/mesh_test.c

```diff
@@ -110,6 +111,12 @@ static void bt_enabled(void)
 		return;
 	}
 
+	err = bt_mesh_cdb_create(test_net_key);
+	if (err) {
+		FAIL("Failed to create CDB (err %d)\n", err);
+		return;
+	}
+
```

Comment:
It'd be better to just do a manual setup for provisioner as well. This could create trouble later